openapi-generator/templates/python/rest.mustache

```diff
@@ -152,6 +152,9 @@ class RESTClientObject(object):
         if 'Content-Type' not in headers:
             headers['Content-Type'] = 'application/json'
 
+        if query_params:
+            query_params = self.flatten_params(query_params)
```

Comment:
I guess this works too, but IMO the proper way would be to do it similar to `parameters_to_tuples` in https://github.com/phrase/openapi/blob/tsi-1763-python-format-options/openapi-generator/templates/python/api_client.mustache#L164 and respecting given style (`deepObject` in our case)

Reply:
That seems to be the better place. But regarding `deepObject` what would you expect there? I don't have this parameter information there anymore. Shouldn't it be enough here to flatten all dicts?

Reply:
Of course. Really thought I've removed these lines before 👀